app/status/views.py

```diff
@@ -5,32 +5,60 @@
 from . import utils
 from ..models import Framework
 from dmutils.status import get_flags
+from app import search_api_client
 
 
 @status.route('/_status')
-def status_no_db():
+def status():
 
     if 'ignore-dependencies' in request.args:
         return jsonify(
             status="ok",
         ), 200
 
     version = current_app.config['VERSION']
+    apis = [
```

Comment:
Yeah probably, but I'd like the keep the code as character-for-character identical as possible to the equiv code in the other repo's, because it'll be easier to compare and move into utils when/if we get around to that bit of technical debt.

Reply:
Makes sense. Merge it 😸